### PR TITLE
fix: Command `rm` remove file with flag `-r`

### PR DESCRIPTION
**Pull request by umaYnit**

### CODE REVIEW

**Comment by umaYnit** [comment]
By the way,  how about support deleting multiple files? like `rm -r a_file a_file_folder`

File: duckscript_sdk/src/sdk/std/fs/rm/mod_test.rs [hunk 1]
```diff
@@ -56,6 +56,22 @@ fn run_path_is_file() {
     assert!(!path.exists());
 }
 
+#[test]
+fn run_path_is_file_and_with_flag() {
+    let path = Path::new("./target/_duckscript/rm/file.txt");
```
Comment (sagiegurari) [inline duckscript_sdk/src/sdk/std/fs/rm/mod_test.rs:61]: can we put a unique file name here so tests running in parallel won't conflict?

**Comment by sagiegurari** [comment]
thanks a lot!!!
as for multiple files, sounds like really good idea but as a separate PR

File: duckscript_sdk/src/sdk/std/fs/rm/mod_test.rs (hunk 1 above)
Comment (umaYnit) [reply duckscript_sdk/src/sdk/std/fs/rm/mod_test.rs:61]: Should be that, I'll change it right away

**Comment by sagiegurari** [comment]
merging now. thanks a lot!!!
i'm going to also add the multi file support and fix the echo issue you raised before i release a new duckscript version.

**Pull request merged**